test/ICSharpCode.SharpZipLib.Tests/Base/InflaterDeflaterTests.cs

```diff
@@ -167,12 +234,20 @@ private void TryManyVariants(int level, bool zlib, RunCompress test, byte[] buff
 		/// </summary>
 		[Test]
 		[Category("Base")]
-		public void InflateDeflateNonZlib()
+		public void InflateDeflateNonZlib([Range(0, 9)] int level)
 		{
-			for (int level = 0; level < 10; ++level)
-			{
-				RandomDeflateInflate(100000, level, false);
-			}
+			RandomDeflateInflate(100000, level, false);
+		}
+
+		/// <summary>
+		/// Basic async inflate/deflate test
+		/// </summary>
+		[Test]
+		[Category("Base")]
+		[Category("Async")]
+		public async Task InflateDeflateNonZlibAsync([Range(0, 9)] int level)
+		{
+			await RandomDeflateInflateAsync (100000, level, false);
```

Comment:
Codacy found an issue: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/piksel/SharpZipLib/pullRequest?prid=5745476)